ufo2fontir/src/source.rs

```diff
@@ -35,6 +26,61 @@ impl DesignSpaceIrSource {
     }
 }
 
+fn glif_files<'a>(
+    ufo_dir: &Path,
+    layer_cache: &'a mut HashMap<String, HashMap<String, PathBuf>>,
```

Comment:
not a blocker, just an observation: depending on how convoluted this gets, I often end up writing a tiny `LayerCache` type here that wraps the hashmap and provides a simple API tailored to my needs.

Reply:
I waffled hard on whether it was justified here. I feel like it's not *quite* but I reserve the right to decide I"m an idiot in future.

Reply:
totally, it's definitely not clear cut either way.